src/pages/global.tsx

```diff
@@ -7,11 +8,44 @@ import { componentSpacing, maxTextWidth } from 'theme/metrics';
 import trpc, { trpcNext } from 'trpc';
 import moment from 'moment-timezone';
 
+type MeetingSlot = {
+  id: number;
+  tmUserId: string;
+  meetingId: string;
+  meetingLink: string;
+  groupId: string | null;
+  updatedAt: string;
+};
+
 export default function Page() {
   const { data } = trpcNext.globalConfigs.get.useQuery();
   const [matchFeedbackEditableUntil, setMatchFeedbackEditableUntil] =
     useState<DateColumn>();
   const [saving, setSaving] = useState(false);
+  const meetingSlotQuery = trpcNext.meetingSlot.list.useQuery();
```

Comment:
define a new react function for the meeting slot UI component. also move exiting code in Page() to a new function for the `matchFeedbackEditableUntil` component.